src/sass/pages/portfoliows.scss

```diff
@@ -0,0 +1,9 @@
+a.dynamic, span.dynamic, div.dynamic, p.dynamic, button.dynamic {
+	display: none!important;
+}
+
+span.loading:before {
+  content: "Loading ...";
```

Comment:
Will remove it as the same loading effect used on https://www.binary.com/trading?l=EN will be used. In general I guess it would be to cumbersome to deal with translation in sass/css.